packages/integration-testsuite/package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@apollo/cache-control-types": "^1.0.2",
     "@apollo/client": "^3.6.9",
+    "@apollo/server": "^4.0.5",
```

Comment:
I think this should be precise rather than caret?

Reply:
[8f7221d](https://github.com/apollographql/apollo-server/pull/7114/commits/8f7221dc713996a1cfd4dc11b21a23b3afc84032)